for-research/public-likelihoods/public-likelihoods.ipynb

```diff
@@ -0,0 +1,936 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3246ef2d-4f29-4082-9d4c-7fe3c0abf61e",
+   "metadata": {},
+   "source": [
+    "# Analysing statistical workspaces with xRooFit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25deda8b-dc32-41f7-80be-4badd58f52a9",
+   "metadata": {},
+   "source": [
+    "# 1. Exploring workspaces\n",
+    "\n",
+    "This notebook covers how to open a workspace and explore its content using the xRooFit API. All workspace exploration involve using the [xRooNode](https://root.cern.ch/doc/master/classROOT_1_1Experimental_1_1XRooFit_1_1xRooNode.html) class, which provides a high-level API for interacting with RooFit/HS3 workspaces."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b5b12609-f8d8-4ff7-aebe-95b81fec7d9b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1mxRooFit -- Create/Explore/Modify Workspaces -- Development ongoing\u001b[0m \n",
+      "                xRooFit : http://gitlab.cern.ch/will/xroofit\n",
+      "                Version: 77eb847 [2024-03-19 10:27:42 +0100]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "'6.31/01'"
```

Comment:
At the moment we are in a bit of a limbo - the software required for the notebook isn't sufficiently up-to-date in ROOT 6.30, so the ROOT nightly has to be used at the moment, or otherwise the updated software has to be built on top. Which is the lesser of two evils?
Once ROOT 6.32 is available (supposed to be coming at end of this month) then that should be able to be used without anything additional required.

Reply:
Ah, got it. I don't have a strong feeling, but I might be inclined to tidy and then release this once 6.32/00 is out so that folks can take everything from a stable release out of the box.

Reply:
I'd generally agree with you too, that we should wait for ROOT 6.32. But I hear from others that people are urgently waiting for these workspaces and can't bare to wait any longer .... others should comment on this (I'm not in a hurry, myself).